lib/tasks/active_stash.rake

```diff
@@ -24,6 +24,12 @@ end
 namespace :active_stash do
   desc "Assess sensitive data used in a Rails application and generate a report"
   task(:assess, [:quiet] => :environment) do |task, args|
+    if !defined?(Rails)
+      raise ActiveStash::RailsUndefinedError, "ActiveStash Assess can currently only be used in Rails projects"
+    end
+
+    Rails.application.eager_load!
```

Comment:
Pulling this Rails-related code up into the Rake task out of `Assess` so that we can test the `Assess` class without Rails.